k8s/deploy.yml

```diff
@@ -41,6 +43,17 @@ spec:
         resources: {}
         terminationMessagePath: /dev/termination-log
         terminationMessagePolicy: File
+      - name: prometheus-exporter
```

Comment:
LGTM.

Prom won't scrape these metrics as is, we'd need to add a few lines around line 4 of this file (as a child of 'metadata' /sibling of 'labels')
```
  annotations:
    prometheus.io/path: /metrics
    prometheus.io/port: 9113
    prometheus.io/scrape: true
```

that should cause the metrics to be read by prom.

Reply:
@duallain done